src/zope/dublincore/browser/metadataedit.py

```diff
@@ -22,6 +22,11 @@
 from zope.i18nmessageid import MessageFactory
 _ = MessageFactory('zope')
 
+try:
+    unicode
+except NameError:  # PY3
+    unicode = str
```

Comment:
I've discovered that I like this idiom, as it avoids the branching:

```suggestion
text_type = type(u'')
```